docs/pages/blog/2020-q1-update.md

```diff
@@ -0,0 +1,110 @@
+
+  - We have collaborated with [Andy Edwards](https://github.com/jedwards1211) to provide [snippets](https://marketplace.visualstudio.com/items?itemName=vscodeshift.material-ui-snippets) with Visual Studio Code:
+
+  ![snippets](/static/blog/2020-q1-update/snippets.gif)
+
+- 💎 We have introduced [Sketch assets](/blog/2020-introducing-sketch/).
+
+  <img src="/static/blog/2020-q1-update/sketch.png" alt="sketch" style="max-width: 160px;" />
+
+But this summary is just scratching the surface. We have accepted 572 commits from 214 different contributors.
+
+## Company
+
+We are thrilled to welcome two new full-time developers on Material-UI:
+
+  - [Dmitriy Kovalenko](https://github.com/dmtrKovalenko), the author of @material-ui/pickers.
+    <img src="https://avatars0.githubusercontent.com/u/16926049" alt="dmitriy" style="max-width: 160px; margin: unset; margin-bottom: 24px; border-radius: 2px;" />
+
+  - [Damien Tassone](https://github.com/dtassone/), an experienced developer in the finance industry.
+    <img src="https://avatars0.githubusercontent.com/u/936978" alt="damien" style="max-width: 160px; margin: unset; margin-bottom: 24px; border-radius: 2px;" />
+
+### Growth between Q4 2019 and Q1 2020
+
+- 📦 From 3.2M to 4.8M downloads per month on npm.
+- ⭐️ From 53.3k to 56.2k stars, leave us yours 🌟.
+- 👨‍👩‍👧‍👦 From 1,581 to 1,720 contributors on GitHub.
+- 💰 Grew monthly financial support by 47%.
+- 🏢 From 3 to 5 full-time equivalent developers, spread among multiple financially supported core team members.
+
+## Our roadmap intent for Q2 2020
+
+We'll do our best, no guarantee!
+
+- 📣 We will analyze and publish the results of the "Material-UI Developer Survey 2020". You haven't contributed to it yet, you can [fill it with this link](https://forms.gle/TYWRdvgyZs4AhZNv8), thanks!
+- 🎨 We will release Figma assets for Material-UI.
+
+  <img src="/static/blog/2020-q1-update/figma.png" alt="figma" style="max-width: 160px; margin-bottom: 24px;" />
+
+- 🇨🇳 We will translate the documentation to Chinese at 100%. We are collaborating with [Danica Shen](https://github.com/DDDDDanica), a native speaker, to complete the effort. So far, we have translated 73% of the documentation and peer-reviewed 39%. You can help us out on [Crowdin](https://translate.material-ui.com/).
+
+  <img src="/static/blog/2020-q1-update/chinese.png" alt="chinese" style="max-width: 134px; margin-bottom: 24px;" />
+
+- 🔥 We will initiate our work on the [next major: v5](https://github.com/mui-org/material-ui/issues/20012).
+You can expect the following:
+
+  - A feature freeze on v4.
+  - The introduction of deprecation messages in the next v4 minors. These messages will help developers upgrade to v5.
+  - A progressive bug fixes freeze on v4, to the exception of security issues and important bugs.
+  - At least 6 months of work on v5 to get to a stable release (probably more). You can follow our [milestone](https://github.com/mui-org/material-ui/milestone/35).
```

Comment:
```suggestion
  - At least 6 months of work on v5 to get to a stable release (probably more). You can follow our [milestones](https://github.com/mui-org/material-ui/milestone/35).
```

Reply:
What's the motivation for the plural?

Reply:
```suggestion
  - At least 6 months of work on v5 to get to a stable release (probably more). You can follow our progress using our [milestone](https://github.com/mui-org/material-ui/milestone/35).
```
```suggestion
  - At least 6 months of work on v5 to get to a stable release (probably more). You can follow our [milestone](https://github.com/mui-org/material-ui/milestone/35) to track our progress.
```

Whoops - you're right. Didn't realised what the sentence was saying.